src/map/date.c

```diff
@@ -77,6 +77,15 @@ int date_get_sec(void)
 	return lt->tm_sec;
 }
 
+// Returns YYYYMMDD of now
+int date_get_date(void)
+{
+	return
+		date_get_year() * 10000 +
```

Comment:
calling 3 times date functions here too slow, each this function will call time(NULL)
better call in once in date_get_date and calculate date from it